kalite/coachreports/api_views.py

```diff
@@ -99,7 +99,12 @@ def learner_logs(request):
     return JsonResponse({
         "logs": output_logs,
         "contents": output_objects,
-        "learners": [learner for learner in learners.values("first_name", "last_name", "username", "pk")],
+        "learners": [{
```

Comment:
Sometimes 'learners' gets collapsed to a list from the Queryset. This insures against that eventuality.